bukkit/src/main/java/net/whimxiqal/journey/bukkit/util/MaterialGroups.java

```diff
@@ -223,6 +223,15 @@ private MaterialGroups() {
   public static boolean isVerticallySpecialPassable(Material material) {
     switch (material) {
       case LADDER:
+      case VINE:
+      case SCAFFOLDING:
+      case WATER:
+      case TWISTING_VINES:
+      case WEEPING_VINES:
+      case CAVE_VINES:
+      case TWISTING_VINES_PLANT:
+      case WEEPING_VINES_PLANT:
```

Comment:
These are nice, but let's add these in a separate PR.

Reply:
Figured it was 50/50 you'd say that, I'll remove it from this.

Reply:
Removed all the `MaterialGroups` changes, I'll likely create a new PR for them though.